releases/ampd/2025-05-20-ampd-v1.8.0.md

```diff
@@ -0,0 +1,76 @@
+# Ampd v1.8.0
+
+|  | **Owner** |
+|-----------|------------|
+| **Created By** | @cjcobb23 <cj@interoplabs.io> |
+| **Deployment** | TBD
+
+| **Network** | **Deployment Status** | **Date** |
+|-------------|----------------------|----------|
+| **Devnet Amplifier** | Deployed | 2025-05-19 |
+| **Stagenet** | - | TBD |
+| **Testnet** | Deployed | 2025-05-20 |
+| **Mainnet** | - | TBD |
+
+
+[Release](https://github.com/axelarnetwork/axelar-amplifier/releases/tag/ampd-v1.8.0)
+
+## Background
+This ampd release makes changes to XRPL support that aim to increase robustness.
+A config change is introduced, affecting the base config as well as XRPL specific config.
+
+## Config change
+
+The basic config, regardless of supported chains, requires some additions. The below sections need to be added:
+```
+[tm_grpc_timeout]
+nanos="0"
+secs="5"
+
+[tofnd_config.timeout]
```

Comment:
@cjcobb23 should this not be `[tofnd_config_timeout]`?

Reply:
no